### ISSUE TITLE
Accessibility - Input fields not functioning with Android TalkBack

### ISSUE THREAD

**Issue by jcmalone** [ionic-team/ionic#10791]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[x] **2.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
When Android TalkBack is enabled and the user selects an input or ion-input, the keyboard will sporadically close and/or ignore user typed letters.

**Expected behavior:**
When Android TalkBack is enabled and the user selects an input or ion-input, the keyboard input should be tracked in the selected field.

**Steps to reproduce:**
Enable TalkBack on an Android device and attempt to fill out a form. 

**Other information:**
1. No issue with Samsung Voice Assistant on input fields
2. Applies to both input and ion-input
3. Workaround is having TalkBack focus on another element after selecting the input field, keyboard input is then able to work.

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Cordova CLI: 6.5.0
Ionic Framework Version: 2.1.0
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.0.0
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Windows 10
Node Version: v6.9.1
Xcode version: Not installed
```

**Comment by jgw96** [comment]
Hello, thanks for using Ionic. We will look into this.
